.github/workflows/compilation_on_android_ubuntu.yml

```diff
@@ -368,9 +368,9 @@ jobs:
           mkdir wasi-libc
           cd wasi-libc
           git init
-          # "Rename thread_spawn import" commit on main branch
+          # "Fix a_store operation in atomic.h" commit on main branch
           git fetch https://github.com/WebAssembly/wasi-libc \
-            8f5275796a82f8ecfd0833a4f3f444fa37ed4546
+            1dfe5c302d1c5ab621f7abf04620fae92700fd22
```

Comment:
Had better also compile wasi-libc in Ubuntu-22.04 instead using the pre-release wasi-sdk-thread version since there is issue in it:
Merge L337 to L346, and remove  L366 and

Reply:
I thought of that, but it would mean that now we remove the usage wasi-sdk 20 pre-release from the ci, basically reverting all the changes previously added in #2021. When wasi-sdk 20 final release comes out we will have to add those changes again.

I was leaving the wasi-sdk 20 part as it is since in practice we didn't encounter problems in the main branch, the hanging issues, only reproducible by running each test many times, always passed unnoticed in the CI.

Reply:
Got it, so let's keep it.